homeassistant/components/websocket_api/commands.py

```diff
@@ -240,7 +240,9 @@ async def handle_call_service(
     except vol.Invalid as err:
         connection.send_error(msg["id"], const.ERR_INVALID_FORMAT, str(err))
     except HomeAssistantError as err:
-        connection.logger.exception(err)
+        connection.logger.error(
```

Comment:
We've been discussing this alternative approach:
https://github.com/home-assistant/core/pull/101037/commits/0dc7960349bdf61dee590ac6d3cff346b4a213f7

Reply:
CC @emontnemery

Reply:
Yeah, that way is more promising👍 
We should perhaps move it to its own PR?